browser/src/app/TilesMiddleware.ts

```diff
@@ -2370,7 +2321,6 @@ class TileManager {
 		if (!tile) return;
 
 		tile.invalidateCount++;
-		tile.allowFastRequest();
```

Comment:
This looks like the most important change here. The rest of the code seems good to me. If you are confident with this line, we can get this in.

Reply:
I'm confident with this as `tile.forceKeyframe` is called just below which in turn calls `allowFastRequest` - The only difference here is not calling `forceKeyframe` on a tile if it's about to be removed (so no difference in practice). At least, that's what I think :)

Reply:
Right, i missed it, thanks for the improvements :)